lib/schemas/descriptor.rb

```diff
@@ -70,7 +70,7 @@ def refresh_schema
       end
 
       if result
-        redis_store!(result.to_a)
+        redis_store!(filter_tables(result.to_a))
         @cache = redis_retrieve
```

Comment:
@dollschasingmen is there a reason why this is not something like
```
redis_store!(thing)
@cache = thing
```
instead of populating the cache from redis?